ts/components/WalletTabIcon.tsx

```diff
@@ -0,0 +1,79 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import { Badge, View } from "native-base";
+import React from "react";
+import { Platform, StyleSheet, Text } from "react-native";
+import { connect } from "react-redux";
+import { getUnreadTransactionsSelector } from "../store/reducers/entities/readTransactions";
+import { GlobalState } from "../store/reducers/types";
+import variables from "../theme/variables";
+import IconFont from "./ui/IconFont";
+
+type OwnProps = {
+  color?: string;
+};
+const MAX_BADGE_VALUE = 99;
+const styles = StyleSheet.create({
+  textBadgeStyle: {
+    fontSize: 10,
+    fontFamily: "Titillium Web",
```

Comment:
Use the makeFontStyle function instead of changing the font and the weight directly.

Reply:
updated within 7918e5770e6ccc309a80032b76d2aed136ad3c46